components/Blog/blogPage.js

```diff
@@ -0,0 +1,27 @@
+import BlogList from './blogList'
+import SideNav from './SideNav'
+import useBlogContent from './useBlogApi'
+import React from 'react'
+
+function useToRenderBlog () {
+  const ALL_SCHOOL = 'All'
+  const blogs = useBlogContent()
+  const [blogCountries, updateSchool] = React.useState([ALL_SCHOOL])
```

Comment:
What does `blogCountries` represent? It's named `blogSchools` on L19. 

Additionally, some comments around this function specifically would help with maintainability. This is a very dense section of the codebase.

Reply:
`blogCountries` was a mistake... Now I changed to `schools`... 
I added a few comments.. 

please look through it and let me know your comments...